src/ontology/mondo-ingest.Makefile

```diff
@@ -92,6 +92,7 @@ $(COMPONENTSDIR)/ordo.owl: $(TMPDIR)/ordo_relevant_signature.txt config/properti
 			--update ../sparql/ordo-construct-subsets.ru \
 			--update ../sparql/ordo-construct-d2g.ru \
 		query --update ../sparql/ordo-replace-annotation-based-mappings.ru \
+		remove --term oboInOwl:hasDbXref --axioms "annotation" \
```

Comment:
@matentzn are you running a build with this update?

Reply:
final data build [PR](https://github.com/monarch-initiative/mondo-ingest/pull/768)